cookiecutter-django-ida/{{cookiecutter.repo_name}}/tox.ini

```diff
@@ -35,57 +35,63 @@ DJANGO_SETTINGS_MODULE = {{ cookiecutter.project_name }}.settings.test
 addopts = --cov {{ cookiecutter.project_name }} --cov-report term-missing --cov-report xml
 norecursedirs = .* docs requirements site-packages
 
+# Filter depr warnings coming from packages that we can't control.
+filterwarnings =
```

Comment:
Non-blocking: Is this not useful for non-IDA cookiecutters?

Reply:
It looks like two of them are likely only a problem in Django worlds, but it wouldn't hurt to squelch them everywhere.